digdag-core/src/main/java/io/digdag/core/database/migrate/Migration_20160926123456_AddDisabledColumnToSchedules.java

```diff
@@ -0,0 +1,13 @@
+package io.digdag.core.database.migrate;
+
+import org.skife.jdbi.v2.Handle;
+
+public class Migration_20160926123456_AddDisabledColumnToSchedules
```

Comment:
Ok. I guess I'll do `disabled_at = now()` instead of generating the timestamp in Java and fighting with pg & h2 timestamp parsing.